src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.cs

```diff
@@ -22,9 +22,22 @@ public sealed partial class TimeZoneInfo
         private const string ZoneTabFileName = "zone.tab";
         private const string TimeZoneEnvironmentVariable = "TZ";
         private const string TimeZoneDirectoryEnvironmentVariable = "TZDIR";
+
+#if !TARGET_BROWSER
```

Comment:
Instead of using `#if` in the code, can we factor the code out into a separate file? ex. `TimeZoneInfo.Unix.MinimalGlobalData.cs` or similar?

That will make it easier to control when it is and isn't included.

Reply:
Sure, I can take that approach.  There's still a lot of shared stuff unrelated to the names though.

Perhaps it would be better to move these things to the existing `TimeZoneInfo.GetDisplayName.cs`, as they are all related to getting display names from full icu?

Reply:
> Perhaps it would be better to move these things to the existing TimeZoneInfo.GetDisplayName.cs, as they are all related to getting display names from full icu?

I think that could work. Just note that in C# code, we tend to shy away from `#if` when possible, and choose to opt for partial files. See https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/project-guidelines.md#conventions-for-forked-code

Reply:
Actually, I like your names better.  Will submit a revision shortly using those instead of `#if`